### ISSUE TITLE
Tokenizer for o1

### ISSUE THREAD

**Issue by texasroh** [openai/tiktoken#363]
`o1` has been released. 
I couldn't get encoding for `o1`
`encoding_for_model('o1')`

`o1` isn't existed in the map.

**Comment by sotayamashita** [comment]
@texasroh 
Might be related?
https://github.com/openai/tiktoken/issues/367

**Comment by Somerandomguy10111** [comment]
Yes #367 is exactly the same issue. I closed it to avoid duplication if anyone is wondering why it happens I posted an explanation in the #367 thread.

**Comment by guvkay20** [comment]
We now have the same issue with o3; I don't see why OpenAI has not solved such a simple issue

**Comment by hauntsaninja** [comment]
This is resolved in tiktoken 0.9
